Plugins/Detray/src/DetrayConversionUtils.cpp

```diff
@@ -0,0 +1,84 @@
+// This file is part of the Acts project.
```

Comment:
Well, not really, because the material we are writing out for the moment is old-style Gen1 material, which is not yet based on Grids, I will update this soon, but the Gen1-type material is the best debugged for the moment.